cvat/apps/engine/models.py

```diff
@@ -193,6 +199,29 @@ class Job(models.Model):
     class Meta:
         default_permissions = ()
 
+    def complete(self):
+        with connection.cursor() as cursor:
+            cursor.execute("""
+                SELECT task.name, image.path
+                FROM engine_segment segment
+                JOIN engine_task task ON segment.task_id = task.id
+                JOIN engine_image image ON task.id = image.task_id AND image.frame = segment.start_frame
+                WHERE segment.id = %s
+            """, [self.segment_id])
+            task_name, image_path = cursor.fetchone()
+
+        _, sequence_name = parse_frame_name(image_path)
+        annotator = self.assignee.username if self.assignee else ''
+        annotation_date = date.today()
+
+        try:
+            client = create_inventory_client()
+            affected_cells = client.record_sequence_completion(sequence_name, task_name, annotator, annotation_date)
+            slogger.glob.info("Job %s completed. Made a record in inventory file: '%s'", self.id, affected_cells)
+        except Exception:
+            slogger.glob.exception("Error while making the job completion record")
```

Comment:
print exception message

Reply:
When `logger.expection()` is called within `except` block, the exception and the traceback are added automatically to the logs:
![error](https://user-images.githubusercontent.com/14096024/91850992-1ec16980-ec67-11ea-806d-a33a64c28007.png)